server/routers/adRouter.js

```diff
@@ -1,9 +1,30 @@
 const router = require('express').Router();
 const controller = require('../controllers');
 
-router.get('/', controller.admin.get);
-router.post('/', controller.admin.post);
-router.put('/', controller.admin.put);
-router.delete('/', controller.admin.delete);
+router.get('/reservations', controller.reservations.get);
+router.post('/reservations', controller.reservations.post);
+router.put('/reservations', controller.reservations.put);
+router.put('/reservations/:guestId', controller.reservations.put);
+router.delete('/reservations', controller.reservations.delete);
+
+router.get('/rooms', controller.rooms.get);
+router.post('/rooms', controller.rooms.post);
+router.put('/rooms', controller.rooms.put);
```

Comment:
Can remove put route to '/rooms'. Room updates will be handled by '/rooms/:room_id'